neard/src/cli.rs

```diff
@@ -554,6 +554,9 @@ pub(super) struct LocalnetCmd {
     /// Whether to configure nodes as archival.
     #[clap(long)]
     archival_nodes: bool,
+    /// Whether to configure nodes to track all shards.
+    #[clap(long)]
+    track_all_shards: bool,
```

Comment:
In what case you wouldn't want this value to be true?
I can think about testing Phase 2 features, such as validators switching from tracking one shard to tracking another shard. But then "not tracking all shards" doesn't provide enough information to express which config I the nodes to have.

What do you think of this suggestion:
- Change the default behavior to create a config tracking all shards
and
- Add a flag to override `tracked_shards` config value. Which can be set to be empty, or to another specific subset of shards.

Reply:
I very much like the suggestion. I made this argument a string with three possible values:
- a comma separated list of shard ids e.g. "0,3"
- "all" to track all shards - it cannot be expressed as comma separated list as it depends on --shards
- "none" to track all shards - it cannot be expressed as comman separated list as passing empty values in cmdline is hard